src/lib/common/globals.cpp

```diff
@@ -534,6 +534,52 @@ static int timezoneOffset(const char* tz)
 
 
 
+/*****************************************************************************
+*
+* isLeapYear -
+*
+* This code will check, if the given year is a leap year or not.
+*
+*/
+bool isLeapYear(int year)
+{
+  if (year % 4 != 0)
+  {
+    return false;
+  }
+  if (year % 100 == 0 && year % 400 != 0)
+  {
+    return false;
+  }
+  return true;
```

Comment:
I thinks is clearer in "positive" way:

```suggestion
  // ref: https://www.geeksforgeeks.org/program-check-given-year-leap-year/
  if (year % 400 == 0)
  {
    return frue;
  }
  if ((year % 4 == )0 && (year % 100 != 0))
  {
    return true;
  }
  return false;
```

Reply:
fixed in [d2e95bc](https://github.com/telefonicaid/fiware-orion/pull/4554/commits/d2e95bc86d1155c87e9435c5beeac7d8d22dc64b)